app/javascript/react/container/StateShowContainer.js

```diff
@@ -24,15 +28,52 @@ class StateShowContainer extends Component {
         .then(state => {
           this.setState({ stateShow: state })
         })
+        .then(this.getParks())
         .catch(error => console.error(`Error in fetch: ${error.message}`));
     }
 
+
+    getParks() {
```

Comment:
Do states have_many parks? If so, please use a *serializer* for state with a have_many with parks to avoid having to do two fetches!

Reply:
We're now using a serializer